src/service.ts

```diff
@@ -129,8 +129,8 @@ function getDeclarationFiles() {
 }
 
 function getInitialCommands() {
-  var codes = getDeclarationFiles().map(dir => `/// <reference path="${dir}" />`)
```

Comment:
Not having a newline at the end here leads to errors if appended text doesn't start with a newline. Try checking out old version of this file and **:load** any .ts file that doesn't start with a newline as a first command to repl.

I know i can just prepend file contents with a newline, but what I did is more robust.